.github/workflows/prepare-release.yml

```diff
@@ -35,6 +35,9 @@ jobs:
           }
 
           validate_stable_version() {
+            if [[ -z "$1" ]]; then
```

Comment:
i would move this check to line 50 to make it easier to find that we're not doing anything with stable if the candidate-stable value is left blank

Reply:
makes sense